docs/updates/migrate-v4.md

```diff
@@ -9,59 +9,85 @@
 
 ## Node.js and npm Version Support
 
-To be defined
+**This release requires Node.js versions v20.11.0, v21.2.0 or higher as well as npm v10 or higher.**
+Support for older Node.js and npm releases has been dropped and will cause an error.
 
 ## Specification Versions Support
 
-To be defined
+Going forward, **only projects with Specification Versions 2.0 and higher are supported.**
+
+If a legacy specification version is detected, **an automatic migration is attempted.**
+Your old projects might therefore still work unless they have a non-standard configuration in their ui5.yaml.
 
 ## Changes for Projects
 
-To be defined
+!!! info
+    ✅ Projects defining **Specification Version 3.x** are expected to be **fully compatible with UI5 Tooling v4**
```

Comment:
```suggestion
    ✅ Projects defining **Specification Version 2.x** are expected to be **fully compatible with UI5 Tooling v4**
```

There shouldn't be any difference between Specification Version 2 and 3 in this regard